components/base-adresse-nationale/voie/numero.js

```diff
@@ -5,19 +5,37 @@ import Link from 'next/link'
 import theme from '@/styles/theme'
 
 import {getNumeroComplet} from '@/lib/ban'
+import Certification from '../certification'
 
-function Numero({id, numero, suffixe, lieuDitComplementNom}) {
+function Numero({id, numero, suffixe, lieuDitComplementNom, certifie}) {
```

Comment:
`certifie` --> `isCertified`

Reply:
C'est l'objet `numero` qui renvoie la propriété `certifie`

Reply:
A corriger dans ce cas, cela doit être un oubli car on préfixe toujours les booléens avec `is` ou `has`.

Reply:
C'est l'API qui renvoie cette propriété dans `address` plus exactement

Reply:
D'accord mais rien n'empêche de renommer la prop de `<Numero />` si `certifie` est bien un booléen renvoyé par l'API.